src/ui/MessageInput/index.jsx

```diff
@@ -137,7 +137,7 @@ const MessageInput = React.forwardRef((props, ref) => {
       //
     }
     setMentionedUserIds([]);
-    setIsInput(textField?.innerText?.trim().length > 0);
+    setIsInput(textField?.textContent?.trim().length > 0);
```

Comment:
Unfortunately jsdom doesn't recognize `innerText` So I had to replace it with `textContent` which is  similar to `innerText`  😕 
https://github.com/jsdom/jsdom/issues/1245

Otherwise we won't be able to add unit tests anymore.

Reply:
give me a little bit of time to test ~

Reply:
Need to test `onPaste` condition

Reply:
How can I test the condition?

Reply:
I wanted to do a smoke test manually haha
If you have free time, maybe add a test to detect XSS -> something like https://dev.to/lexlohr/unit-testing-with-jest-typescript-hands-on-session-345o#the-second-task

Reply:
@sravan-s  So you're trying to test `onPaste` condition -> want me to add a test to detect XSS. I don't get what you really wanted to do 😅 Could you give me more detail what makes you think we need XSS test here?

Reply:
Had a short huddle with Sravan 🎧 
 - XSS test is not related to this change but kind of suggestion which is  good to have: probably will do separately
 - There could be some edge cases hardly caught by unit tests so manual tests is also needed

Reply:
Attaching this link too https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innertext for someone who might wonder